src/plugins/tables/tables.js

```diff
@@ -293,7 +293,7 @@ $document.on( "submit", ".wb-tables-filter", function( event ) {
 				$value = ( $fData ) ? $fData : "-0";
 				$regex = "(" + $value.replace( /&nbsp;|\s/g, "\\s" ).replace( /\$/g, "\\$" ) + ")";
 			}
-		} else if ( $elm.is( "input[type='date']" ) ) {
+		} else if ( $elm.is( "input[type='date']" ) && $elm.val() ) {
```

Comment:
Should we add the same logic for "select" (line 243), "number" (line 255) and "Checkbox" (line 329) ? Do we will experience the same issue as it was reported?

Reply:
Currently the only effect this issue has is it makes the date required for any filter to work if the date filter is used

As for the select and checkbox they well only fail if HTML is coded incorrectly

I can add fail safes for that if you wish